core/src/main/resources/hudson/PluginManager/table.jelly

```diff
@@ -147,6 +147,7 @@ THE SOFTWARE.
                 <span style="margin-left:2em;"></span>
               </j:if>
               <f:submit value="${%Download now and install after restart}" />
+              <st:include page="check.jelly"/>
```

Comment:
Does this not also show it on the _Installed_ page, where it does not make much sense?